src/i18n/locales/en/translation.json

```diff
@@ -3,18 +3,19 @@
     "welcome": {
       "title": "Welcome to Alby",
       "set_password": {
-        "title": "Set an unlock passcode",
-        "description": "This passcode will allow you to unlock Alby on this device. It cannot be recovered, but may be changed later.",
+        "title": "Set extension unlock passcode",
+        "description1": "Set a passcode to unlock Alby Extension in this browser",
+        "description2": "Unlock passcode cannot be recovered, but may be changed ",
         "choose_password": {
           "label": "Choose an unlock passcode:"
         },
         "confirm_password": {
           "label": "Let's confirm you typed it correct:"
         },
         "errors": {
-          "enter_password": "Please enter a passcode.",
-          "confirm_password": "Please confirm your passcode.",
-          "mismatched_password": "Passcodes don't match."
+          "enter_password": "Please enter a password.",
```

Comment:
Inconsistent. Why password here?

Reply:
as per figma

Reply:
![image](https://github.com/getAlby/lightning-browser-extension/assets/55848322/aa95574b-7d7d-4dcf-943b-736e35f76165)

Reply:
sorry, my omission. should be passcode there